source/documentation/guidance/buildpacks.md

```diff
@@ -1,44 +1,120 @@
+				<td style="vertical-align: middle; text-align: center; background-color: #ffbf47">Yours</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					App admin
+					<br> - ensure app instances are up and running
+					<br> - ensure provisioning capacity
+          <br> - stream application logs
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					<a href="https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service">Backing services</a>
+					<br> - maintain service availability
+					<br> - apply security patches
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+			<tr>
+				<td style="vertical-align: middle; text-align: left">
+					Physical hardware
+					<br>- resolve hardware failures
+					<br>- encrypt network data traffic
+				</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+				<td style="vertical-align: middle; text-align: center; background-color: #2b8cc4">PaaS</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+### Standard buildpacks
+
+You are only responsible for the app code and its dependencies. This includes [penetration testing](https://docs.cloud.service.gov.uk/#penetration-testing) your app.
 
-#### Standard buildpack
-You are only responsible for the app code and its dependencies.
+If you're relatively new to cloud or have a small development team, we recommend using the standard buildpacks to maximise the support you will receive from GOV.UK PaaS.
 
-#### Custom buildpacks
+### Custom buildpacks
 
-You are responsible for managing this custom buildpack as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
+You are responsible for managing this custom buildpack's language and runtime, as well as the app code and its dependencies. This applies both in setting up the app, and running it in production.
 
 When setting up the app:
 
 - Do your own due diligence on finding a maintained and supported custom buildpack so that you have a community to call upon in the case of problems
-- Test deploying with the buildpack early in your build process, not just before go-live
+- Test deploy with the buildpack early in your build process, not just before go-live
```

Comment:
I think `deploying` was better so we don't put two verbs after each other.

Reply:
/cc @jonathanglassman

Reply:
I would go with `Test deploy` as it is more active than `Test deploying`.

Reply:
ok